tools/Az/Az.psd1

```diff
@@ -66,7 +66,7 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.2.5'; },
                @{ModuleName = 'Az.CognitiveServices'; RequiredVersion = '1.8.0'; }, 
                @{ModuleName = 'Az.Compute'; RequiredVersion = '4.9.0'; }, 
                @{ModuleName = 'Az.ContainerInstance'; RequiredVersion = '1.0.3'; }, 
-               @{ModuleName = 'Az.ContainerRegistry'; RequiredVersion = '2.2.0'; }, 
+               @{ModuleName = 'Az.ContainerRegistry'; RequiredVersion = '2.2.1'; }, 
```

Comment:
Please check if you need to change the version in Az. 

Try below steps
1. Keep all changes except reverting Az.psd1.
2. `dotnet msbuild build.proj /t:"Build" /p:"Configuration=Release"`
3. `.\RunVersionController.ps1 -Release "February 2021"`

I'd like to know the result of Az.psd1.  Can it pick up the latest version of ContainerRegistry? Can it add its release note to release note of Az?

Reply:
> Please check if you need to change the version in Az.
> 
> Try below steps
> 
> 1. Keep all changes except reverting Az.psd1.
> 2. `dotnet msbuild build.proj /t:"Build" /p:"Configuration=Release"`
> 3. `.\RunVersionController.ps1 -Release "February 2021"`
> 
> I'd like to know the result of Az.psd1. Can it pick up the latest version of ContainerRegistry? Can it add its release note to release note of Az?

After tried these steps, version of Az.ContainerRegistry in Az.psd1 is still 2.2.0